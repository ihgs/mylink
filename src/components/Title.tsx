

const Title = ({title}:{title: string}) =>{
  
  return (
    <h1 className="text-3xl font-bold underline m-10">
      {title}
    </h1>
  )

}

export default Title;