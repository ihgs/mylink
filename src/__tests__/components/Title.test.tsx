import { render, screen } from "@testing-library/react";
import Title from "../../components/Title";

test("[snapshot]Title", () => {
  const view = render(<Title title="testtittle" />);
  expect(view.container).toMatchSnapshot();
});
