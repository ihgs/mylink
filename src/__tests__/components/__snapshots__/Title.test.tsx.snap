// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[snapshot]Title 1`] = `
<div>
  <h1
    class="text-3xl font-bold underline m-10"
  >
    testtittle
  </h1>
</div>
`;
